automation/deploy_hco.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+set -ex
+
+HCO_VERSION=1.4.0
+CMD=oc
+NS=kubevirt-hyperconverged
+
+# Create a catalog source
+cat <<EOF | ${CMD} apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: hco-catalog-source
+  namespace: openshift-marketplace
+spec:
+  sourceType: grpc
+  image: "quay.io/kubevirt/hyperconverged-cluster-index:${HCO_VERSION}"
+  displayName: Kubevirt Hyperconverged Cluster Operator
+  publisher: Kubevirt Project
+EOF
+
+# Create the kubevirt-hyperconverged namespace
+cat <<EOF | ${CMD} apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: "${NS}"
+EOF
+
+# Create operator group
+cat <<EOF | ${CMD} apply -f -
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+    name: kubevirt-hyperconverged-group
+    namespace: "${NS}"
+EOF
+
+# Create Subscription
+cat <<EOF | ${CMD} apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+    name: hco-operatorhub
+    namespace: "${NS}"
+spec:
+    source: hco-catalog-source
+    sourceNamespace: openshift-marketplace
+    name: community-kubevirt-hyperconverged
+    channel: "${HCO_VERSION}"
+EOF
+
+# Wait for the deployments to be available
+"${CMD}" wait -n "${NS}" deployment hco-operator --for=condition=Available --timeout="1080s"
+"${CMD}" wait -n "${NS}"  deployment hco-webhook --for=condition=Available --timeout="1080s"
+
+# Create the HyperConverged resource
+curl "https://raw.githubusercontent.com/kubevirt/hyperconverged-cluster-operator/release-1.4/deploy/hco.cr.yaml" | ${CMD} -n "${NS}" apply -f -
```

Comment:
HCO_VERSION is a variable set to 1.4.0, but here the link to the CR is hard-coded.
For sake of consistency and not missing this one when version is promoted, would you prefer to extract `1.4` into a variable as well ?

Reply:
I agree with Moti.
Something like
```
RELEASE_BRANCH=release-${HCO_VERSION%.*}
```
and
```
 curl "https://raw.githubusercontent.com/kubevirt/hyperconverged-cluster-operator/${RELEASE_BRANCH}/deploy/hco.cr.yaml" | ${CMD} -n "${NS}" apply -f -

Reply:
Done + add DVs